app/src/main/java/ingsw/group1/findmyphone/location/GeoPosition.java

```diff
@@ -0,0 +1,150 @@
+package ingsw.group1.findmyphone.location;
+
+import android.location.Location;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+/**
+ * A class holding a Geographical position, in the form latitude and longitude.
+ *
+ * @author Riccardo De Zen
+ */
+public class GeoPosition {
+
+    static final String POSITION_SPLIT_SEQUENCE = "@";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/RicDeZen/ingsw-group1-findmyphone/pullRequest?prid=4950325)